tests/stub/optimizations/test_optimizations.py

```diff
@@ -25,7 +27,7 @@ def tearDown(self):
     def test_pull_pipelining(self):
         def test():
             script = "pull_pipeline{}.script".format("_tx" if use_tx else "")
-            self._server.start(path=self.script_path(script),
+            self._server.start(path=self.script_path("v4x3", script),
```

Comment:
Why does the pull pipelining is now only available for the 4.3 scenario?

Reply:
It was like this previously. I just moved files.